delay

```diff
@@ -0,0 +1,1135 @@
+src/java/javax/websocket/HandshakeResponse.java:    public static final String SEC_WEBSOCKET_ACCEPT = "Sec-WebSocket-Accept";
+src/java/javax/websocket/server/HandshakeRequest.java:    static final String SEC_WEBSOCKET_KEY = "Sec-WebSocket-Key";
+src/java/javax/websocket/server/HandshakeRequest.java:    static final String SEC_WEBSOCKET_PROTOCOL = "Sec-WebSocket-Protocol";
+src/java/javax/websocket/server/HandshakeRequest.java:    static final String SEC_WEBSOCKET_VERSION = "Sec-WebSocket-Version";
+src/java/javax/websocket/server/HandshakeRequest.java:    static final String SEC_WEBSOCKET_EXTENSIONS= "Sec-WebSocket-Extensions";
+src/java/nginx/unit/Context.java:                if (tag_name.equals("async-supported")) {
```

Comment:
Hi Zhidao, something seems to have gone very wrong here...

```
git show b4326dd20e4f229b5bff67a367f11ef007224db8 --summary --stat
commit b4326dd20e4f229b5bff67a367f11ef007224db8
Author: Zhidao HONG <z.hong@f5.com>
Date:   Wed Jan 31 14:08:57 2024 +0800

    Var: refactored nxt_var_ref_get().

 delay         | 1135 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 src/nxt_var.c |   14 +-
 2 files changed, 1143 insertions(+), 6 deletions(-)
 create mode 100644 delay
```

This `delay` file seems have been erroneously added, then I think I see it being deleted in another commit...